airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -46,6 +47,7 @@ Features
 * ``Use Boto waiters instead of customer _await_status method for RDS Operators (#27410)``
 * ``Handle transient state errors in 'RedshiftResumeClusterOperator' and 'RedshiftPauseClusterOperator' (#27276)``
 * ``Add retry option in RedshiftDeleteClusterOperator to retry when an operation is running in the cluster (#27820)``
+* ``Deprecate 'full_url_mode' for SecretsManagerBackend; whether a secret is a JSON or URL is inferred (#27920)``
```

Comment:
I thought this changes not include to 6.2.0 and still in dev

Reply:
The `full_url_mode` kwarg is deprecated, but not removed (in the sense that it won't return an error).

https://github.com/apache/airflow/blob/68acca53f4c9b91cdaae9af02234da15a9a735dd/airflow/providers/amazon/aws/secrets/secrets_manager.py#L125

There is basically zero harm in removing it, since anyone who is using all URLs will not be disrupted by its removal under normal circumstances (their secrets will continue to parse normally), and also anyone who is using all JSONs will not be disrupted (their secrets will also continue to parse normally).

Reply:
Oh sorry, revisiting. You're talking about how I put it under the wrong section (6.2.0 instead of 7.0.0)? Yes. I will fix that.

Reply:
Yeah, I meant this feature changelog was about unreleased feature.